alembic/versions/718efd6cbfc1_test_migration.py

```diff
@@ -0,0 +1,34 @@
+"""Test migration
```

Comment:
While this was probably a test, I think we would want this to be something like `create_user_table` in the file name and in this comment.

Reply:
I think we probably want to drop this migration entirely, I just asked Adam to show me what a migration might look like.

Alternatively, once I've put some basic models together, we could merge them into this PR so that the alembic etc. integration all goes in with the default models.

Reply:
Thank you, was missing that part of context